packages/office-ui-fabric-react/src/components/Slider/Slider.tsx

```diff
@@ -133,20 +136,17 @@ export class Slider extends BaseComponent<ISliderProps, ISliderState> implements
                 ref='thumb'
                 className={ css('ms-Slider-thumb', styles.thumb) }
                 { ...ariaLabel ? { 'aria-label': ariaLabel } : {} }
-                style={ getRTL() ?
-                  { 'right': thumbOffsetPercent + '%' } :
-                  { 'left': thumbOffsetPercent + '%' } }
+                style={ this._getThumbStyle(vertical, thumbOffsetPercent) }
               />
-              <span className={ css('ms-Slider-active', styles.activeSection) } style={ { 'width': thumbOffsetPercent + '%' } }></span>
-              <span className={ css('ms-Slider-inactive', styles.inactiveSection) } style={ { 'width': (100 - thumbOffsetPercent) + '%' } }></span>
+              <span className={ css('ms-Slider-active', styles.lineContainer, styles.activeSection) } style={ !vertical ? { 'width': thumbOffsetPercent + '%' } : { 'height': (thumbOffsetPercent) + '%' } }></span>
```

Comment:
more repeat...

const lengthString = vertical ? 'height' : 'width';
then later, use that:

```tsx
<span style={ { [lengthString]: thumbOffsetPercent + '%' } } ... />
```

Reply:
Thank you so much. Are there any other issues I can fix?